### DISCUSSION TITLE
Upload to github

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#4] [General]
Hi,

Can you please upload https://techstudent11.herokuapp.com to github? I would like to make some changes to it, if you want! 

Thanks,
Kendall Does Coding

**Comment by TechStudent10** [comment]
@KendallDoesCoding yeah I should. The only reason I can't do it is because it's on my old laptop and I'll have to dig through it in order to find it.

**Comment by KendallDoesCoding** [comment]
Oh

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Can you make a new repo, and add me as a contributor and I will try to add the files.

**Reply by TechStudent10** [reply]
Sure, you can try. Hold on a min.

**Reply by KendallDoesCoding** [reply]
Thanks, please add me as a contributor, I see you made the repo

**Reply by KendallDoesCoding** [reply]
Thanks!

**Comment by KendallDoesCoding** [comment]
Done, Please remove contributor access, I will make my edits, pull requests
so you can check them out. If you agree, you'll need to login to your
heroku app and link this github repo to the website, I've done that for all
my websites on my domain.

**Reply by TechStudent10** [reply]
Noted.

**Comment by KendallDoesCoding** [comment]
Check out all the files I added here -
https://github.com/TechStudent11/heroku-portfolio

**Comment by KendallDoesCoding** [comment]
Actually, can you give me contributor access back, I'm creating some edits
in the 2nd branch

**Reply by TechStudent10** [reply]
Oh shoot sorry.

**Comment by KendallDoesCoding** [comment]
And will create pull requests for your approval

**Comment by KendallDoesCoding** [comment]
Nah, it's ok, I first told you to remove the contributors role but then I
was doing something.

Also, hope that's all the files, hope you know how I got the code too

**Reply by TechStudent10** [reply]
Inspect Element

**Reply by TechStudent10** [reply]
Also, I just remembered. I made this a Flask app. So oops. I'll make the changes later.

**Reply by TechStudent10** [reply]
For now, this doesn't matter.

**Comment by KendallDoesCoding** [comment]
Yeah!

**Comment by KendallDoesCoding** [comment]
Oh...

**Comment by KendallDoesCoding** [comment]
Can you make this github a github pages repo too, if possible?

**Comment by KendallDoesCoding** [comment]
Actually, ill upload it to netlify and deploy it

**Reply by TechStudent10** [reply]
I don't have a Netlify account soooooo...

**Comment by KendallDoesCoding** [comment]
Yeah, netlify makes me rename everything to index.html, so deploy it on
github pls

**Reply by TechStudent10** [reply]
I would rather use Heroku for this.

**Reply by TechStudent10** [reply]
But first I need to make it a Flask App. So I will deploy it when I'm done.

**Comment by KendallDoesCoding** [comment]
ok, either is fine, just saying so we can see our edits.

**Comment by KendallDoesCoding** [comment]
Ok, can you temp make it a github site then and then delete it when done
with heroku?

**Reply by TechStudent10** [reply]
Yeah sure I guess.

**Comment by KendallDoesCoding** [comment]
Thanks!

**Reply by TechStudent10** [reply]
https://techstudent11.github.io/heroku-portfolio/

gives 404 for some reason

**Reply by TechStudent10** [reply]
oh wait nevermind

**Comment by KendallDoesCoding** [comment]
thanks

**Comment by KendallDoesCoding** [comment]
oh wait.. here too you need to name everything index.html

**Comment by KendallDoesCoding** [comment]
ill just open with live server

**Reply by TechStudent10** [reply]
K. Make sure to post screenshots.

**Comment by KendallDoesCoding** [comment]
Yeah ofc, I gotta go to sleep soon though.. Will work on this in the morning

**Reply by TechStudent10** [reply]
Oh, wait what time is it where you live? It's 2 PM in the afternoon here!

**Comment by KendallDoesCoding** [comment]
12:33am

**Reply by TechStudent10** [reply]
Oh shoot! You should **really** go to bed now. It's already Monday where you live. **MONDAY!**

**Comment by KendallDoesCoding** [comment]
I commited, welcome to my portfolio, overwriting welcome to my homepage, is
that fine or do you want it to be  welcome to my homepage?

**Comment by KendallDoesCoding** [comment]
Yeah lol

**Comment by KendallDoesCoding** [comment]
I think, instead of being a separate static folder for homepage, we just
put styling there, not in any folder for the homepage

**Comment by KendallDoesCoding** [comment]
Also, like my website does, display your projects:

https://kendalldoescoding.gq/

Even ottomated does that, in a better way then me:
https://ottomated.net/

We got the code to both mine and ottomated, so we can work on doing that
for yours too.

**Comment by KendallDoesCoding** [comment]
About page:
[image: image.png]

**Comment by KendallDoesCoding** [comment]
Please delete the github deployment, as you know, it's not working properly.

**Comment by KendallDoesCoding** [comment]
@TechStudent11 lets work on a projects tab for the portfolio, did you
finish the setup for the heroku website?

**Reply by TechStudent10** [reply]
nope not yet sorry. I'm gonna have to do that over the weekend (it's Monday)

**Comment by KendallDoesCoding** [comment]
Ok, that's all good!

**Comment by KendallDoesCoding** [comment]
Can I make a new branch, naming all the files in the folders index.html?

**Comment by KendallDoesCoding** [comment]
@TechStudent11/techstudent11 ***@***.***>  I need
your help, can you help me make this doodle only embed, iframe on this
website, I don't want anything else, I only want the top part to embed

Google Doodle URL with info:
https://www.google.com/doodles/celebrating-pizza
Website to embed on: https://kendalldoescoding.gq/celebratingpizza

Thanks,
Kendall Does Coding

**Reply by TechStudent10** [reply]
I don't get what you're asking for.

**Reply by TechStudent10** [reply]
@KendallDoesCoding check my status

**Comment by KendallDoesCoding** [comment]
Oh ok, you can do it whenever you're free, if you know how to :)

**Comment by KendallDoesCoding** [comment]
You see, that is a full webpage, but I want to iframe only this much

[image: image.png]

I also need to block the scroll wheel if possible...

Thanks,
Kendall Does Coding

**Reply by TechStudent10** [reply]
@KendallDoesCoding Since you are using email, I can't see the images. Can you please come to the discussion: https://github.com/TechStudent11/techstudent11/discussions/4

**Reply by KendallDoesCoding** [reply]
https://gm1.ggpht.com/ai3BEMXZasPl889G-tbBbfVi4cfUorQGEo_ERGFOjpHrWwjp5UlLHE9yuwyLEe4t4jMiRgX6-szZlPdMYQNzTESN03b64DqwgUhmYfSaycoy_OLUg0oVFetgGCZi5_B0yoHepqCmqNrxVi91qAFPweekh4z7MayMVmOZsqQynKf4PGrqInk6HGMDxed74TB5udsdfye2hqe8PxVb623wMK5GWj44w-0_RNl1Lt5aShjgFu4G_sPhKTKD8_MnhfltLLDojJO20qFBFdDyD1WW7EeH_-6gejmUgciR02IKkXswXKG8PLMuUB0SgVwua3dwL7OeOROVB48PfjfZsWaXYzTZATvx2EB2vZRYA_ohEgoHFvDKLKY9qyKXHW1pTtkilKGUjmhIh4fCslSnzAJH2o3Z-89B7KkxjH4Jq9EyuhZyj5AGbtnZR5zUU9Ui8dr0Q1mhulPke3y3XT_Tqckw8iuakL3miZbOms0nSLO7ZulJ9RycQs7vT-SzpQSAykbmS3W-W1MWrxGQkicwqbSQmpPXAwE3fopdpX4a_2vN47M8P8-IsOfXGTT0XzmVVoAQ7T8xCX0wCAG3ZjIn6ZZK6TqAkuVJUc_Z6POA-o3n4TojSsdYX0Mk2XVnJHICl4k4MECtmvmuVD2yfPzcwikyl9U-kncvqFT5K_jnWARlWbv1mX5QKz-jJ11iNL81LWryx57uqUERlWAzL7u7WKrOJrQOs6jcFoQU_pDv51_D=s0-l75-ft-l75-ft

**Reply by KendallDoesCoding** [reply]
Img url

**Reply by TechStudent10** [reply]
![image](https://user-images.githubusercontent.com/76978184/145026205-dc99b908-98d8-4f68-b485-095cd8eeca82.png)

**Comment by KendallDoesCoding** [comment]
Hey TechStudent11,

I added the portfolio here for the time being -
https://techstudent11.kendalldoescoding.gq/

**Reply by TechStudent10** [reply]
Thanks

**Comment by KendallDoesCoding** [comment]
Although, the about and contact pages aren't formatted idk what that's about

**Reply by TechStudent10** [reply]
Yeah, that's weird.

**Comment by KendallDoesCoding** [comment]
Oh ok, I'll send a screenshot on github then.

**Comment by KendallDoesCoding** [comment]
![image](https://user-images.githubusercontent.com/84712013/145027719-8514d744-deb2-4cef-bce8-cb350264e10d.png)

**Reply by KendallDoesCoding** [reply]
Did that come through @TechStudent11

**Reply by TechStudent10** [reply]
yep it did

**Comment by KendallDoesCoding** [comment]
Cool, so do you know how I can iframe only that part onto that page?

**Reply by TechStudent10** [reply]
nope, not entirely sure.

**Comment by KendallDoesCoding** [comment]
Ok, np then . I'll test some things.

**Comment by KendallDoesCoding** [comment]
Hi, Can you get down to adding that repo to heroku app and also please do this - https://devcenter.heroku.com/articles/github-integration and set all deploys to auto come to the site.

Thanks,
KDC
github@kendalldoescoding.gq

**Reply by TechStudent10** [reply]
yeah maybe later

i'll let you know when i do

**Comment by KendallDoesCoding** [comment]
Ok
